packages/react-native/React/Views/RCTView.m

```diff
@@ -1318,6 +1318,14 @@ - (void)displayLayer:(CALayer *)layer
   [self updateClippingForLayer:layer];
 }
 
+#if TARGET_OS_OSX // [macOS
+- (void)updateInternalReactTransform:(CATransform3D)transform
+{
+  [self setTransform3D:transform];
+  [self setNeedsDisplay];
```

Comment:
Do we need this line? If stuff works without it, I'm not opposed to removing it. Not blocking though.

Reply:
I don't see anything immediately wrong if we remove it, but considering that [it was originally present](https://github.com/microsoft/react-native-macos/blob/80a1f9e7782f9a96c6607460270830a48f28946b/packages/react-native/React/Views/RCTViewManager.m#L250), I'd rather not remove it here.